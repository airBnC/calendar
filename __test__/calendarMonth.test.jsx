import React, { Component } from 'react';
import CalendarMonth from '../client/src/components/calendarMonth.jsx';
import styled from 'styled-components';
import { shallowToJson } from 'enzyme-to-json';
import { shallow } from 'enzyme';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
Enzyme.configure({ adapter: new Adapter() });
import CalendarDay from '../client/src/components/calendarDay.jsx';
const thisMonth = 6;
const monthChanged = {
	daysOpen: [1,2,3,5,6,7,8],
	beginning: 6,
	day: 31

}
describe('CalendarMonth', () => {
  it('should render correctly', () => {
    const output = shallow(
      <CalendarMonth currentData={monthChanged} />
    );
    expect(shallowToJson(output)).toMatchSnapshot();
  });
});