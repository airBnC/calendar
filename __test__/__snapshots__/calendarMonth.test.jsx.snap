// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CalendarMonth should render correctly 1`] = `
<styled.div>
  <styled.div>
    <styled.span>
      Sun
    </styled.span>
    <styled.span>
      Mon
    </styled.span>
    <styled.span>
      Tues
    </styled.span>
    <styled.span>
      Weds
    </styled.span>
    <styled.span>
      Thurs
    </styled.span>
    <styled.span>
      Fri
    </styled.span>
    <styled.span>
      Sat
    </styled.span>
  </styled.div>
  <styled.div>
    <CalendarDay
      key="0"
    />
    <CalendarDay
      key="1"
    />
    <CalendarDay
      key="2"
    />
    <CalendarDay
      key="3"
    />
    <CalendarDay
      key="4"
    />
    <CalendarDay
      key="5"
    />
    <CalendarDay
      day={1}
      key="6"
    />
    <CalendarDay
      day={2}
      key="7"
    />
    <CalendarDay
      day={3}
      key="8"
    />
    <CalendarDay
      cost=""
      day={4}
      key="9"
    />
    <CalendarDay
      day={5}
      key="10"
    />
    <CalendarDay
      day={6}
      key="11"
    />
    <CalendarDay
      day={7}
      key="12"
    />
    <CalendarDay
      day={8}
      key="13"
    />
    <CalendarDay
      cost=""
      day={9}
      key="14"
    />
    <CalendarDay
      cost=""
      day={10}
      key="15"
    />
    <CalendarDay
      cost=""
      day={11}
      key="16"
    />
    <CalendarDay
      cost=""
      day={12}
      key="17"
    />
    <CalendarDay
      cost=""
      day={13}
      key="18"
    />
    <CalendarDay
      cost=""
      day={14}
      key="19"
    />
    <CalendarDay
      cost=""
      day={15}
      key="20"
    />
    <CalendarDay
      cost=""
      day={16}
      key="21"
    />
    <CalendarDay
      cost=""
      day={17}
      key="22"
    />
    <CalendarDay
      cost=""
      day={18}
      key="23"
    />
    <CalendarDay
      cost=""
      day={19}
      key="24"
    />
    <CalendarDay
      cost=""
      day={20}
      key="25"
    />
    <CalendarDay
      cost=""
      day={21}
      key="26"
    />
    <CalendarDay
      cost=""
      day={22}
      key="27"
    />
    <CalendarDay
      cost=""
      day={23}
      key="28"
    />
    <CalendarDay
      cost=""
      day={24}
      key="29"
    />
    <CalendarDay
      cost=""
      day={25}
      key="30"
    />
    <CalendarDay
      cost=""
      day={26}
      key="31"
    />
    <CalendarDay
      cost=""
      day={27}
      key="32"
    />
    <CalendarDay
      cost=""
      day={28}
      key="33"
    />
    <CalendarDay
      cost=""
      day={29}
      key="34"
    />
    <CalendarDay
      cost=""
      day={30}
      key="35"
    />
    <CalendarDay
      cost=""
      day={31}
      key="36"
    />
    <CalendarDay
      key="37"
    />
    <CalendarDay
      key="38"
    />
    <CalendarDay
      key="39"
    />
    <CalendarDay
      key="40"
    />
    <CalendarDay
      key="41"
    />
  </styled.div>
</styled.div>
`;
